instrument/utils/utilities.py

```diff
@@ -0,0 +1,446 @@
+"""
+Auxilary HKL functions.
+
+.. autosummary::
+    ~setaz
+    ~freeze
+    ~show_constraints
+    ~set_constraints
+    ~change_diffractometer
+    ~plotselect
+    ~set_counting_time
+    ~set_experiment
+    ~list_functions
+    ~read_delta
+    ~transfocator
+
+"""
+
+import gi
+
+gi.require_version("Hkl", "5.0")
+from gi.repository import Hkl
+from hkl.user import (
+    _check_geom_selected,
+    current_diffractometer,
+    select_diffractometer,
+)
+
+from instrument.collection import RE, diffract, fourc, scaler, counters
+from instrument.collection import psic
+from inspect import getmembers, isfunction
+from polartools import (
+    hkl_utils,
+    load_data,
+    diffraction,
+    absorption,
+    pressure_calibration,
+    process_images,
+    area_detector_handlers,
+    manage_database,
+)
+from apstools import utils
+
+from hklpy.hkl import user, util
+from hkl.util import Constraint
+
+from numpy import loadtxt, interp
+
+import fileinput
+import pathlib
+import sys
+
+path = pathlib.Path("startup_experiment.py")
```

Comment:
Where is this going to import from? I think it will, right now, look for it from the current "utils" folder.